pkgs/data/icons/fluent-icon-theme/default.nix

```
@@ -0,0 +1,74 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, gtk3
+, hicolor-icon-theme
+, jdupes
+, roundedIcons ? false
+, blackPanelIcons ? false
+, colorVariant ? [ ]
+,
+}:
+let
+  pname = "Fluent-icon-theme";
+in
+lib.checkListOfEnum "${pname}: available color variants" [
+  "standard"
+  "green"
+  "grey"
+  "orange"
+  "pink"
+  "purple"
+  "red"
+  "yellow"
+  "teal"
+  "all"
+]
+  colorVariant
+  stdenvNoCC.mkDerivation
+rec {
```

```suggestion
] colorVariant

stdenvNoCC.mkDerivation rec {
```

I am pretty sure this is a bug or we are doing something weird here